build/configs/imxrt1050-evk/loadable_elf_apps/defconfig

```diff
@@ -309,9 +309,9 @@ CONFIG_ARCH_USE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="128,128,128,128,256,256,"
-CONFIG_FLASH_PART_TYPE="kernel,none,bin,bin,bin,bin,"
-CONFIG_FLASH_PART_NAME="kernel,imxrt,micom,micom,wifi,wifi,"
+CONFIG_FLASH_PART_SIZE="256,256,256,256,256,256,"
+CONFIG_FLASH_PART_TYPE="none,none,bin,bin,bin,bin,"
```

Comment:
We need to set a type for kernel because we use 2 partitions for kernel with switching.
So it is registered as ftl type in configure_partitions.

Reply:
Ok.. i will change it back.